dist/package.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
are you sure `dist` folder should be in git repo?

Reply:
Yes, it is. It contains sources from external libraries like axios and `@actions/github`. Unfortunately, GitHub actions doesn't run yarn install